src/snippets/templates/ts/klasaEvent.ts

```diff
@@ -0,0 +1,28 @@
+import { Event, EventStore, KlasaClient } from 'klasa';
+
+export default class extends Event {
+
+  constructor(client: KlasaClient, store: EventStore, file: string[], directory: string) {
```

Comment:
Just FYI, `client` is gone in the alpha and settings branches, keeping this will make this outdated very quickly.